Xamarin.Forms.Core/Device.cs

```diff
@@ -16,15 +16,15 @@ public static class Device
 
 		public static TargetPlatform OS { get; internal set; }
 
-		internal static DeviceInfo Info
+		public static DeviceInfo Info
```

Comment:
The team discussed this a couple of weeks ago.. we aren't sure this api is ready to be public, is not future proof, what if we have different form factors, and we want not orientation changes but something like screen changes.. maybe that+s more  future proof that only treat orientation.